setup.py

```diff
@@ -412,18 +412,18 @@ def getExtensionModules(nupicCoreReleaseDir, platform, bitness, cmdOptions=None)
 
   extensions = []
 
-  libDynamicCppRegion = Extension(
-    "nupic." + getLibPrefix(platform) + "cpp_region",
+  libDynamicPyRegion = Extension(
+    "nupic." + getLibPrefix(platform) + "py_region",
```

Comment:
@david-ragazzi: There are additional references to `cpp_region` in setup.py that you may have missed. Please check for other references to `cpp_region` in this and related projects.